### PR TITLE
including methods to compute Industrial Costs

### PR DESCRIPTION
**Pull request by tarikorra**
Hey guys, I've added methods to compute industrial costs.
Please, review the scripts so we can merge.

regards,
Tarik

### CODE REVIEW

File: regression/scripts/industrial_costs/industrial_costs.py [hunk 1]
```diff
@@ -0,0 +1,510 @@
...
+        manufact_costs.difficulty_factor = 1.5           # (1.0 for conventional, 1.5 for moderately advanc. tech., 2 for agressive use of adv. tech.)
+        manufact_costs.cad_factor        = 0.8           # (1.2 for learning, 1.0 for manual, 0.8 for experienced)
+        manufact_costs.stealth           = 0.0           # (0 for non-stealth, 1 for stealth)
+        manufact_costs.material_factor   = 1.5           # (1 for conventional Al, 1.5 for stainless steel, 2~2.5 for composites, 3 for carbon fiber)
+        manufact_costs.aircraft_type     = 'commercial'  # ('military','general aviation','regional','commercial','business')
+
+
+    return config
+#==================================
+def check_results(new_results,old_results):
```
Comment (planes) [inline regression/scripts/industrial_costs/industrial_costs.py]: This looks like it needs to be cleaned up.

File: trunk/SUAVE/Methods/Costs/Correlations/Industrial_Costs/compute_industrial_costs.py [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+
+"""
+
+	# Unpack
+    costs                   = vehicle.costs.industrial
+    reference_year          = costs.reference_year
+    total_production        = costs.production_total_units
+    n_prototypes            = costs.prototypes_units
+    development_total_years = vehicle.costs.industrial.development_total_years
+    ac_type 		        = costs.aircraft_type.lower()
```
Comment (planes) [inline trunk/SUAVE/Methods/Costs/Correlations/Industrial_Costs/compute_industrial_costs.py]: just fix the tabulation

**Comment by tarikorra** [comment]
Done.

**Comment by planes** [comment]
Looks good! Thanks Tarik!

**Pull request merged**